misc/buildtestdata.sh

```diff
@@ -33,6 +33,8 @@ mkdir -p "${E2EDATA}"
 
 (cd "${E2EDATA}" && tar -j -c -f net_done.tar.bz2 --exclude node.log --exclude agreement.cdv net)
 
+if false; then
```

Comment:
Was this supposed to be committed? Looks like a debugging thing.

Reply:
This is part of moving to a new e2e validation script. The data build gets smaller and simpler. Moving from e2etest.py which compares algod sqlite to indexer to e2elive.py which compares algod API to indexer which should be more correct and more sustainable in the future.